### PR TITLE
Changed Poliastro Logo position 

### PR DESCRIPTION
**Pull request by DhruvSondhi**
Hello, 
This PR aims to correct the position of the logo as seen on the front page of the website https://blog.poliastro.space/index.html
Thanks :)

### CODE REVIEW

**Comment by DhruvSondhi** [comment]
Screen Shot as promised/needed :)
image

File: content/pages/home.rst [hunk 1]
```diff
@@ -1,14 +1,19 @@
-Welcome to poliastro webpage
-============================
+Welcome to poliastro
+====================
 
-:title: Welcome to poliastro webpage
+:title: Welcome to Poliastro
```
Comment (astrojuanlu) [inline content/pages/home.rst]: poliastro is not capitalized :)

**Comment by astrojuanlu** [comment]
Merged, thanks!

File: content/pages/home.rst (hunk 1 above)
Comment (DhruvSondhi) [reply content/pages/home.rst]: Awesome :) ... Thanks :)

**Pull request merged**